Templates/RemoteRepo/Template/repo.json

```
@@ -0,0 +1,13 @@
+{
+    "repo_name":"${Name}",
+    "repo_uri": "${RepoURI}",
+    "repo_path": "${RepoPath}",
+    "schemaVersion": "1.0.0",
+    "origin":"Origin for the ${Name} Gem Repository",
```

Lets remove the word `Gem` because this no longer needs to be a Gem repository.
```suggestion
    "origin":"Origin for the ${Name} Remote Repository",
```